composer.json

```diff
@@ -36,7 +36,8 @@
 	},
 	"autoload-dev": {
 		"psr-4": {
-			"Utils\\": "utils"
+			"CodeIgniter\\": "tests/system/",
```

Comment:
This makes me uneasy. Why is it needed?

Reply:
If I will not include this then `CodeIgniter\Models` namespace will not be found and generally all test classes will not be found.

Reply:
Since Composer's and PHPUnit's autoloaders rely on PSR4, the new namespace `CodeIgniter\Models` will be looked on `system/Models/` but that directory doesn't exist. Including `CodeIgniter\\ => tests/system` as a dev namespace will ensure that autoloaders will find the test classes.

Reply:
I guess since it is only on `dev` it should not affect anything else.